### DISCUSSION TITLE
[Feature] View map as interactive globe

### DISCUSSION THREAD

**Discussion by dominikheinz** [immich-app/immich#6738] [Feature Request]
### The feature

Immich's current map implementation utilizes a basic 2D projection format.
I think it would be a nice addition, if there was an option to view the map as an interactive globe.
Not only would it look great, but it might also give a better sense of how far certain locations are apart, aka it is easier to see if an image was taken "on the other side of the world".

Here is an example, what I mean (taken from Google Maps):

[globe.webm](https://github.com/immich-app/immich/assets/32770080/474d4a84-c2a2-4e84-880b-5caf442a0842)

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Maplibre currently doesn't support this, but it is being worked on: https://github.com/maplibre/maplibre/discussions/161

**Comment by bo0tzz** [comment]
This is now available
